youtube_dl/extractor/francetv.py

```diff
@@ -371,12 +371,13 @@ def _real_extract(self, url):
                 self.url_result(dailymotion_url, DailymotionIE.ie_key())
                 for dailymotion_url in dailymotion_urls])
 
-        video_id, catalogue = self._search_regex(
-            (r'id-video=([^@]+@[^"]+)',
+        video_id = self._search_regex(
+            (r'player\.load[^;]+src:[^"\']*["\']([^"\']+)',
```

Comment:
Only whitespace is possible between src: and id.

Reply:
I changed the pattern to only match whitespace there.
```r'player\.load[^;]+src:\s*["\']([^"\']+)'```